validator/src/main/java/com/amazon/aoc/validators/TraceValidator.java

```diff
@@ -140,8 +140,7 @@ private Map<String, Object> getTrace() throws Exception {
       traceFilter += " AND (annotation.aws_local_service = \"local-root-client-call\" OR annotation.aws_local_service = \"local-root-client-call:80\")" ;
     } else {
       traceFilter += (String.format(" AND annotation.aws_local_operation = \"%s %s\"",
-              validationConfig.getHttpMethod().toUpperCase(),
-              validationConfig.getHttpPath()));
+              validationConfig.getHttpMethod().toUpperCase(), validationConfig.getHttpPath()));
```

Comment:
Doesn't seem to be a necessary changes in this PR.

Reply:
Changed back to original format.